coderd/workspaces.go

```diff
@@ -43,6 +43,19 @@ var (
 	errDeadlineBeforeStart = xerrors.New("new deadline must be before workspace start time")
 )
 
+// @Summary Get workspace metadata
+// @ID get-workspace
+// @Security CoderSessionToken
+// @Produce json
+// @Tags Workspaces
+// @Param id path string true "Workspace ID"
```

Comment:
does swaggo not support the uuid type?

Reply:
Fixed, it isn't a type but a format, so I highlighted that. Thanks!